pkg/build/controller/build/build_controller.go

```diff
@@ -1014,6 +1016,25 @@ func (bc *BuildController) handleActiveBuild(build *buildapi.Build, pod *v1.Pod)
 	return update, nil
 }
 
+func isOOMKilled(pod *v1.Pod) bool {
+	if pod.Status.Reason == "OOMKilled" {
```

Comment:
you have constant for this right?

Reply:
I thought since that constant is in `builds` context, it shouldn't be used to check the status reason for `pods` (they currently have the same string describing it but potentially this could change, maybe?)

Reply:
yeah technically they are two different constants.  One is the "build failure reason" and the other is the "pod status reason" so i agree that keeping them as separate constants makes sense.

Reply:
it's too bad there's no k8s constant for this that we can reference.